quart_prometheus_logger.py

```diff
@@ -66,14 +68,15 @@ def __init__(self, app: Optional[Quart] = None, metrics_endpoint: str = "root"):
         self._collectors: Dict[str, MetricType] = {}
         self._custom_labeler: Optional[Callable[["LocalProxy"], Dict[str, str]]] = None
         self._custom_label_names: List[str] = []
+        self._registry = CollectorRegistry()
```

Comment:
I'm fine with no longer using an explicitly-managed registry, but we should ensure this is identically instantiated. As of now, it is missing the `auto_describe` parameter.

Additionally, if folks want to manage it externally, I think it'd be useful to add this as an optional argument.

Reply:
that's a good idea